libs/shared/ui/src/lib/components/navigation-left/navigation-left-sub-link/navigation-left-sub-link.tsx

```diff
@@ -1,48 +1,42 @@
 import { type ReactNode, useEffect, useState } from 'react'
-import { Link, useLocation } from 'react-router-dom'
+import { useLocation } from 'react-router-dom'
 import Icon from '../../icon/icon'
-import { type NavigationLeftLinkProps } from '../navigation-left'
+import { LinkContent, type NavigationLeftLinkProps, linkClassName } from '../navigation-left'
 
 export interface NavigationLeftSubLinkProps {
-  linkContent: (link: NavigationLeftLinkProps) => ReactNode
-  link: NavigationLeftLinkProps
-  linkClassName: (pathname: string, url?: string, badge?: string) => string
+  link: NavigationLeftLinkProps & {
+    subLinks: {
+      title: string
+      url: string
+      badge?: string
+    }[]
+  }
+  children: ReactNode
 }
 
-export function NavigationLeftSubLink(props: NavigationLeftSubLinkProps) {
-  const { link, linkClassName, linkContent } = props
+export function NavigationLeftSubLink({ link, children }: NavigationLeftSubLinkProps) {
   const { pathname } = useLocation()
+  const isActivePath = link.subLinks.some((currentLink) =>
+    linkClassName(pathname, currentLink.url)?.includes('is-active')
+  )
+  // By default only the active path is open
+  const [open, setOpen] = useState(isActivePath)
```

Comment:
It's `true` by default we have all sub-links open, should be `false` 

<img width="1102" alt="Screenshot 2024-09-11 at 17 18 14" src="https://github.com/user-attachments/assets/40f45de5-c62a-4bbc-bbbb-35cbcb94807d">

Reply:
Fixed, I missed only one condition
https://github.com/Qovery/console/pull/1660/commits/97cd36bc0c8fe40ac1b911772e7e4760e0ec69fc
https://github.com/Qovery/console/pull/1660/commits/2bfca482906b9bf7e13199385a4ee9202e37a31d